pages/reference/OS/secure-boot-and-full-disk-encryption/generic-x86-64-gpt.md

```diff
@@ -0,0 +1,146 @@
+
+- **Persist UEFI settings for the expected lifetime of the device**
+
+    As a necessary part of the provisioning process, customers will almost certainly change some UEFI settings to non-default values. The customer should test that these settings will persist for the expected lifetime of their device. For instance, customers should use an RTC / CMOS battery that will endure through the expected lifetime of the device. Note that extended periods without AC / mains power will shorten the life of an RTC battery.
+
+## Setup
+
+There are two steps required to install a secure boot enabled and disk encrypted system:
+
+- Opt-in secure boot mode in the {{ $names.os.lower }} installer
+- Configure the device’s Unified Extensible Firmware Interface (UEFI) firmware for secure boot and setup mode
+
+Note that {{ $names.os.lower }} currently does not support updating from a non-secure boot enabled system into a secure boot enabled one. The only way to install a secure boot and disk encrypted system at this moment is by using a {{ $names.os.lower }} installer image.
+
+
+### Configuring {{ $names.os.lower }} image to opt-in secure boot mode
+
+`Generic x86_64 (GPT)` {{ $names.os.lower }} images ship with the capability to provision a device with secure boot and full disk encryption. While the feature was introduced in {{ $names.os.lower }} `v2.114.21`, it has since been has been hardened for security and updated for broader device compatibility in version `v6.6.1`. {{ $names.company.upper }} recommends that all customers use the most recent releases.
+
+
+#### Using {{ $names.cloud.lower }}
+
+1. Add a device to your fleet. You should be presented with the following new option:
+
+<img alt="Add a device modal with secure boot and full disk encryption modal" src="/img/common/sb-fde/x86/add-device-modal-sb-fde-x86.webp" width="100%">
+
+2. As mentioned, there are [important considerations and caveats][considerations-caveats] when enabling secure boot and full disk encryption. You will be presented with the following disclaimer when enabling the feature:
+
+<img alt="Warning for considerations when enabling secure boot and full disk encryption" src="/img/common/sb-fde/x86/sb-fde-x86-disclaimer.webp" width="100%">
+
+ You can optionally surpress the disclaimer by selecting "Don't show me this warning again for this device type".
+
+ You can then proceed to download the configured image.
+
+ #### Using {{ $names.cli.lower }} 
+
+{{ $names.cli.upper }} versions `16.2.0` or newer allow you to [configure](https://docs.balena.io/reference/balena-cli/#os-configure-image) a {{ $names.os.lower }} installer image to opt-in secure boot by using the following command:
+
+````
+balena os configure <image> --secureBoot --version <version> --fleet <fleetName> --device-type generic-amd64
+
+````
+
+### Flash the image to a USB key
+
+The configured image should be flashed to a USB key.
+
+- Insert the USB key to the host machine.
```

Comment:
In general I think we should refer to the developers machine as "Your laptop", I think host machine is way to confusing and I would like for us to move a way from that across the whole docs. cc @vipulgupta2048 , even though not everyone is using a laptop, that is a clear indication that his means insert it into your computer not the host device you are trying to provision.